### PR TITLE
Improve docs about contributing upstream for generated content

### PR DESCRIPTION
**Pull request by sftim**
- Get ready for a switch to "main"
- Use the current actual release as an example

### CODE REVIEW

**Comment by jimangel** [comment]
Is this ready to come out of draft?

File: content/en/docs/contribute/generate-ref-docs/contribute-upstream.md [hunk 1]
```diff
@@ -91,7 +91,7 @@ will be different in your situation.
 
 Here's an example of editing a comment in the Kubernetes source code.
 
-In your local kubernetes/kubernetes repository, check out the master branch,
+In your local kubernetes/kubernetes repository, check out the primary branch,
```
Comment (kbhawkey) [inline content/en/docs/contribute/generate-ref-docs/contribute-upstream.md:94]: 👋 @sftim .
Could this sentence continue to refer to the `master` branch but indicate that these instructions (prefer or give preference to) will refer to this branch as the primary branch?

Comment (sftim) [reply content/en/docs/contribute/generate-ref-docs/contribute-upstream.md:94]: Yep, that could work.

**Pull request merged**